src/term/mod.rs

```diff
@@ -940,27 +940,27 @@ impl ansi::Handler for Term {
 
     #[inline]
     fn move_backward_tabs(&mut self, count: i64) {
-        err_println!("[unimplemented] move_backward_tabs: {}", count);
+        warn!("[unimplemented] move_backward_tabs: {}", count);
     }
 
     #[inline]
     fn move_forward_tabs(&mut self, count: i64) {
-        err_println!("[unimplemented] move_forward_tabs: {}", count);
+        warn!("[unimplemented] move_forward_tabs: {}", count);
     }
 
     #[inline]
     fn save_cursor_position(&mut self) {
-        err_println!("[unimplemented] save_cursor_position");
+        warn!("[unimplemented] save_cursor_position");
     }
 
     #[inline]
     fn restore_cursor_position(&mut self) {
-        err_println!("[unimplemented] restore_cursor_position");
```

Comment:
These being `err_println!` was a mistake. The can also be replaced with `trace!`.